avaframe/tests/test_distanceTimeAnalysis.py

```diff
@@ -0,0 +1,168 @@
+''' Tests for module distanceTimeAnalysis  '''
+import numpy as np
+import numpy.ma as ma
+import pandas as pd
+import pathlib
+import configparser
+import matplotlib.pyplot as plt
+
+# Local imports
+import avaframe.ana5Utils.distanceTimeAnalysis as dtAna
+import avaframe.in2Trans.ascUtils as IOf
+from avaframe.in3Utils import cfgUtils
+
+
+def test_getRadarLocation():
+    """ test get radar location """
+
+    # setup required inputs
+    cfg = configparser.ConfigParser()
+    cfg['GENERAL'] = {'radarLocation': '10.|25.|40.|57.'}
+
+    # call function to be tested
+    radarFov = dtAna.getRadarLocation(cfg)
+
+    assert radarFov[0] == [10., 25.]
+    assert radarFov[1] == [40., 57.]
+    assert radarFov == [[10., 25.],[40., 57.]]
+
+
+def test_setDEMOrigin():
+    """ test setting of DEM origin back to original """
+
+    # setup required inputs
+    headerDEM = {'xllcenter': 0.0, 'yllcenter': 0.0, 'cellsize': 2., 'ncols': 8, 'nrows': 11}
+    demSims = {'header': headerDEM, 'originalHeader': {'xllcenter': 1.0, 'yllcenter': 5.,
+        'cellsize': 2., 'ncols': 8, 'nrows': 11},
+        'rasterData': np.zeros((11, 8))}
+
+    # call function to be tested
+    demOriginal = dtAna.setDemOrigin(demSims)
+
+    assert demOriginal['header']['xllcenter'] == 1.0
+    assert demOriginal['header']['yllcenter'] == 5.0
+    assert demOriginal['header']['ncols'] == 8
+    assert demOriginal['header']['nrows'] == 11
+    assert demOriginal['header']['cellsize'] == 2.
+
+
+def test_rotate():
+    """ test rotate a vector by an angle """
+
+    # setup required data
+    locationPoints = [[0., 0.], [0., 10.]]
+    theta = 45.
```

Comment:
same here, 45 is maybe not the best idea

Reply:
also use different angle now for testing